specification.md

```diff
@@ -1274,14 +1281,12 @@ to the trigger/produced event.
 
 #### Error Definition
 
-Error definitions define runtime errors that can occur during workflow execution and how to handle them. For more information,
-see the [Workflow Error Handling section](#Workflow-Error-Handling).
-
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
-| expression | JsonPath expression. Should be evaluated against error data. True if expression result is not empty | string | yes |
-| [errorDataFilter](#error-data-filter) | Error data filter definition | object | yes |
-| [transition](#Transitions) | Next transition of the workflow when `expression` is true | object | yes |
+| error | Domain-specific error name, or '*' to indicate all possible errors | string | yes |
+| code | Error code. Can be used in addition to the name to help runtimes resolve to technical errors/exceptions. Should not be defined if error is set to '*' | string | no |
```

Comment:
I think a code, e.g. `code: 500`, can always be used on its own. People probably distinguish error cases by the repsonses, so e.g. my service could reply 500 Server Internal Error and attach an error object (as e.g. in this [Swagger component object example](https://swagger.io/specification/#components-object)). If I want to catch all the errors, should I specify `code: 500` AND `error: *` or just `code: 500`. My intuition tells me that I should omit `error` if I want to catch all of them.

Reply:
i think if you want to catch all 500 errors, you can still define lets say "error": "Internal Server Errors" and have code lets say 500 or whatever the runtime mapping requires.
The error property is the actual portable one so it should be required